.github/workflows/ci.yml

```diff
@@ -182,3 +186,75 @@ jobs:
           helm repo update
           touch kubeconfig && chmod 600 kubeconfig && echo "${{ secrets.KUBECONFIG }}" > kubeconfig
           KUBECONFIG="kubeconfig" helm --namespace="${NAMESPACE}" upgrade "${RELEASE}" jupyterhub/jupyterhub --version=0.11.1 --values config.yaml --cleanup-on-fail
+
+
+  deploy-marketplace:
+
+    needs: [build-docker-image]
+    if: github.ref == 'refs/heads/main'
+
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    environment:
+      name: marketplace
+      url: https://materials-marketplace.aiidalab.net/
+    env:
+      NAMESPACE: default
+      RELEASE: aiidalab-marketplace
+      aiidalab_default_apps: >-
+        aiidalab-widgets-base@https://github.com/aiidalab/aiidalab-widgets-base@stable
+        quantum-espresso@https://github.com/aiidalab/aiidalab-qe@stable
+      aiidalab_factory_reset: ""
+      contact_email: simon.adorf@epfl.ch
+      https_enabled: true
+      image_name: ${{ github.repository }}
+      image_tag: ${{ needs.build-docker-image.outputs.tag }}
+      jh_host: materials-marketplace.aiidalab.net
+
+      marketplace_host: staging.the-marketplace.eu
+      oauth_client_id: "04366497-2d3f-4cc4-8c1a-b6a4a7fa68b7" 
+      oauth_client_secret: ${{ secrets.OAUTH_CLIENT_SECRET }}
+      secret_token: ${{ secrets.SECRET_TOKEN }}
+      oauth_scope: "['email', 'profile', 'openid', 'offline']"
+      admin_users: "['jusong yu', 'simon epfl']"
+
+    steps:
+      -
+        uses: actions/checkout@v2
+      -
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      -
+        name: Install Python requirements
+        run: |
+          pip install \
+            j2cli[yaml] \
+            ruamel.yaml
+          pip freeze
+      -
+        name: Configure deployment
+        run: |
+          j2 .github/workflows/config-template.yaml.j2 -e "" > config.yaml
+
+      # For test: cat the config.yaml
+      - name: Read config.yaml
+        id: cat_jh_config
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./config.yaml
+      - name: Echo config.yaml
+        run: echo "${{ steps.cat_jh_config.outputs.content }}"
```

Comment:
Do we really need this? It has the potential to leak secrets.

Reply:
Sure, it's only for test. Removed.